docs/community/com-startup-connect.md

```diff
@@ -25,15 +25,25 @@ FeatureBase does not currently run on Mac Safari
 
 ## How do I start the FeatureBase Community server?
 
-1. Login to the system
-2. Open a terminal window and cd to the folder
-2. Execute these commands in a terminal window:
+* Open a terminal
+* CD to the `~/featurebase/opt` directory
```

Comment:
we are asking user to CD here and run 2 commands which include cd again.. we are doing same step twice